### ISSUE TITLE
return filepicker_image_url with json response

### ISSUE THREAD

**Issue by denodster** [Ink/filepicker-rails#119]
I'm using a rails backend with a javascript front end, and the two communicate with each other via javascript. In any case we aren't using views (we do use active admin and views for some other admin tools)

I have the need to pass an image url to javascript from the back end. Preferably a collection of them, we used to return security policies via json, but we've found that this feature has been removed in the new version of the gem. Therefore I'm trying to find a way to make filepicker_image_url serve the same function.

I would like to return an array of image urls via JSON and have each one have a valid url. Our current implementation is a model method that appends a security policy to the filepicker_url that is called via the methods argument in .to_json. Therefore I would like to call filepicker_image_url inside of a model method.

TL/DR: Is there a way to call filepicker_image_url in a model method? If there is no way to do this, what is the recommended way of returning urls from the back end via JSON?

**Comment by maurogeorge** [comment]
You can include the `FilepickerRails::ApplicationHelper` module in your model. Example

```ruby
class Peson

   include FilepickerRails::ApplicationHelper
end
```

**Comment by denodster** [comment]
Thank you, this worked well.
